web/modules/custom/portland_migrations/migrations/auditor_blog.yml

```diff
@@ -0,0 +1,38 @@
+id: auditor_blog
+label: 'Auditor Blog Import'
+migration_group: auditor
+source:
+  plugin: csv
+  path: modules/custom/portland_migrations/sources/auditor_blog_news.csv
+  header_row_count: 1
+  keys:
+    - CONTENT_ID
+process:
+  uid:
+    plugin: default_value
+    default_value: 1
+  title: CONTENT_NAME
+  field_summary: SUMMARY_TEXT
+  field_body_content/value: TEXT
```

Comment:
In order to migrate embedded body content images into media items, this field should use the `migrate_body_content_and_linked_media` plugin. Here's an example from the pbot_news migration:
```
field_body_content/value:
    plugin: migrate_body_content_and_linked_media
    source: TEXT
    group_id: 14
```
You'll just need to change the group_id to the ID of the auditor's group so it knows where to save the image media entities it creates.